CHANGELOG.md

```diff
@@ -1,13 +1,23 @@
-# CHANGELOG
+# Changelog
+
+CARTO.js is a JavaScript library that interacts with different CARTO APIs. It is part of the CARTO Engine ecosystem.
+
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```

Comment:
This is kind of cool ☝️

Reply:
:+1: https://github.com/CartoDB/cartodb.js/commit/017188520d9daf89d2a1b5e0a2f3dccbf0f98530